lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -733,8 +733,8 @@ def define
             end
           end
 
-          if ceremony_country == 'italy' && resident_of != 'uk'
-            phrases << :required_supporting_documents_italy
+          if resident_of != 'uk' && data_query.phrase_exists?("required_supporting_documents_#{ceremony_country}")
```

Comment:
I think this is something you mentioned the other day. It seems reasonable given the current state of the code, but more generally I do wonder whether it's adding yet another place where such logic exists.

It feels to me as if the code would be easier to follow if all the characteristics of a given country were represented in a single place, rather than having list of countries and exceptions. But that's a job for another day.